open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb

```diff
@@ -39,7 +39,7 @@
   },
```

Comment:
<p>This is an open source project, instead of a course.</p><p> </p><p>I think you have contribute content as well. So this project only contributes to the major part of this section. Could you please use this <code>which inspires the majority of the content in this chapter</code> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/161/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>